src/main/java/seedu/duke/Parser.java

```diff
@@ -77,14 +77,13 @@ public static Command checkAction(String inputCommand) throws DukeExceptions {
                     projectDescription += inputs[i];
                 }
                 commandType = new ProjectCommand(projectDescription);
-                break;
             } else {
-                System.out.println("Not in Project View!"); //----------REPLACE WITH EXCEPTION
+                throw new DukeExceptions("Add Task"); // REPLACED WITH EXCEPTION
```

Comment:
It is hard to tell from the messageType the error. Example Naming: AddTaskWrongView